lib/src/revset.rs

```diff
@@ -1523,6 +1545,18 @@ where
             operation,
             candidates,
         } => folder.fold_at_operation(operation, candidates)?,
+        RevsetExpression::WithinReference {
+            candidates,
+            commits,
+        } => {
+            let candidates = folder.fold_expression(candidates)?;
+            let commits = commits.clone();
```

Comment:
nit: for consistency, inline here or un-inline on line 1339

Reply:
I copied the style of the surrounding code. Maybe I choose un-inlined version here because it looked better in binary expression nodes.

Reply:
Leaving as is makes sense then. I only noticed the context lines shown by default.